scene/2d/camera_2d.cpp

```diff
@@ -410,6 +410,7 @@ void Camera2D::make_current() {
 	} else {
 		get_tree()->call_group_flags(SceneTree::GROUP_CALL_REALTIME, group_name, "_make_current", this);
 	}
+  _update_scroll();
```

Comment:
Hello there, you should correct indentation of the _update_scroll() to match the previous closing bracket, preferably using a tabulator.

Reply:
As it turns out, I still had two space tabs turned on in Xcode.  It all looks fine in there, switching it to normal tabs fixed the indentation.